cartographer/mapping_3d/local_trajectory_builder.h

```diff
@@ -40,10 +40,24 @@ namespace mapping_3d {
 // without loop closure.
 class LocalTrajectoryBuilder {
  public:
+  // The following two structs are used for representing the result of adding
```

Comment:
Instead of describing where it is used, rather add the comment next to the public member function, i.e. before `AddRangeData()`. For example:
```
  // Returns 'MatchingResult'  when range data accumulation completed,
  // otherwise 'nullptr' .
```